test/onnx/test_caffe2.py

```diff
@@ -34,11 +34,11 @@
 from model_defs.lstm_flattening_result import LstmFlatteningResult
 from model_defs.rnn_model_with_packed_sequence import RnnModelWithPackedSequence
 
-import onnx
-import caffe2.python.onnx.backend as c2
+# import onnx
+# import caffe2.python.onnx.backend as c2
 
 from test_pytorch_common import skipIfTravis, skipIfNoLapack, skipIfNoCuda
-import verify
+# import verify
```

Comment:
Why did those get commented? Did you want to remove them?

Reply:
Mistake. Currently the only way for me to try to run the alexnet test since I don't have onnx installed correctly.